docs/configuration.rst

```diff
@@ -20,17 +20,17 @@ Job configurations
 
 The configuration options for individual jobs are usually specified on
 the command-line, and accessed through the
-`files.get_allargv_config()` function.  The
-`files.get_allargv_config()` function builds a single configuration
+:code:`files.get_allargv_config()` function.  The
+:code:`files.get_allargv_config()` function builds a single configuration
```

Comment:
another option is to use cross-references, e.g.:

    The :py:func:`~impactlab_tools.files.get_allargv_config` function...

This would provide a hyperlink to the corresponding documentation while preserving a fixed-width format. The tilde at the start of the link hides the absolute path elements and just displays the function name.